src/main/scala/nl.knaw.dans.easy.archivebag/EasyArchiveBag.scala

```diff
@@ -82,7 +81,14 @@ object EasyArchiveBag extends Bagit5FacadeComponent with DebugEnhancedLogging {
     if (response.getStatusLine.getStatusCode != HttpStatus.SC_CREATED) throw new IllegalStateException("Error trying to add bag to index")
   }
 
-  private def getBagSequence(bagId: BagId)(implicit ps: Parameters): Try[Option[String]] = Try {
+  private def handleRefBags(versionOfId: BagId)(implicit ps: Parameters): Try[Unit] = {
```

Comment:
IMO a misnomer. This function does not handle ref bags it writes a refBagTxt.

Reply:
but it also fetches the bag sequence from easy-bag-index.
I'll go for `createRefBagsTxt`